CHANGELOG.md

```diff
@@ -4,18 +4,28 @@ All notable changes to this project will be documented in this file.
 ---
 ## [5.0.0](https://github.com/ReactiveX/RxSwift/releases/tag/5.0.0)
 
-* RxSwift 5 is a source-compatible release targeting the Swift 5 compiler.
-* Xcode 10.2 is the minimum supported version (or Swift 5 on Linux).
-* Changes the return type of `ObservableType.toArray` to `Single`. 
-* Adds `compactMap`.
-* Adds `Completable.zip` (alias of `Completable.merge`).
+RxSwift 5 is a _mostly_ source-compatible release targeting the Swift 5 compiler.
+
+**Xcode 10.2 is the minimum supported version (or Swift 5 on Linux).**
 
 If you're using Xcode 10.1 and below, please use [RxSwift 4.5](https://github.com/ReactiveX/RxSwift/releases/tag/4.5.0).
 
+* Relays have been moved to a separate framework - RxRelay, and can be used without RxCocoa. #1924
+* `TimeInterval` has been deprecated in favor of `DispatchTimeInterval`. For example `throttle(1.2)` would change to `throttle(.seconds(1.2))` or `throttle(.milliseconds(1200))`. #1472
```

Comment:
It's not possible to use `.seconds(1.2)`. The value has to be an integer.

Reply:
Fixed, thanks.